finger-exercises-react/react/tic-tac-toe/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_BASE_URL=http://localhost:3005
```

Comment:
The `.env` file should not be committed.
Add it to the `.gitignore` and delete it.